data/language/en-GB.txt

```diff
@@ -3576,7 +3576,7 @@ STR_6125    :Object type
 STR_6126    :Unknown type
 STR_6127    :File: {STRING}
 STR_6128    :The file could not be loaded as some of the objects referenced in it are missing or corrupt. A list of these objects is given below.
-STR_6129    :Copy selected item to clipboard
+STR_6129    :Copy item to clipboard
 STR_6130    :Copy entire list to clipboard
```

Comment:
Perhaps 'Copy entire list to clipboard' should be shortened as well. It barely fits in English, but I think both Dutch and German will be in trouble for this string, too.

Reply:
Shortening it in English does not solve any problems for Dutch and English. Texts should not be translated literally, they should make the idea behind the message clear. (Also, AFAIK, the current Dutch does fit.)

Reply:
I don't disagree, but the localisation aspect was not the main point I was trying to make. The three buttons are equally wide, but their strings are not — especially with two being overly full. I'm hoping to get some balance back.

Reply:
Perhaps we should change the buttons to `Copy`, `Copy all` and `Download all`, then, with the current button texts as tooltips.

Reply:
Done